.github/workflows/publish-devfile-schema.yaml

```diff
@@ -0,0 +1,71 @@
+name: Publish Devfile Schema
+
+# Triggers the workflow on push events but only for the main branch
+on:
+  push:
+    branches: [main]
+
+# Only allow one job of this action to be ran at a time
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish-json-schema:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Checkout devfile docs
```

Comment:
could you rename this step, no longer called devfile docs, probably devfile-web?